nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/processors/hive/PutHiveStreaming.java

```diff
@@ -182,6 +182,7 @@
                     + "Please see the Hive documentation for more details.")
             .required(false)
             .addValidator(HiveUtils.createMultipleFilesExistValidator())
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
```

Comment:
I think we are missing some EL evaluation in the code where this property is used. (L428 for instance)

Reply:
Oops.. Extremely sorry.. My mistake. Cherry-picked the wrong commit. Let me fix this real quick.